src/utils/editor/expression.js

```diff
@@ -64,6 +64,11 @@ export function previewExpression({
   }
 
   if (variables.has(tokenText)) {
+    let variableKey = variables.get(tokenText);
```

Comment:
I think we should try writing a test for this: 

here is the test file: https://github.com/peterschussheim/debugger.html/blob/dont-show-undef-latest/src/utils/tests/expression.js#L5-L31